webpack.config.js

```diff
@@ -35,10 +35,12 @@ module.exports = ( env ) => {
     } ),
     output: {
       filename: ifProd('[name].min.js', '[name].js'),
-      path: ifTest(
-        resolve( context, 'tmp', 'tests' ),
-        resolve( context, 'packages', env.element, 'dist' )
-      ),
+      path:
```

Comment:
I'm not big fan of this kind of indirection this way you can read webpack config without jumping between helper functions